content/applications/marketing/survey/survey_essentials.rst

```diff
@@ -0,0 +1,301 @@
+
+Options tab
+===========
+
+Under the *Options* tab, there are different sections of settings you can modify.
+
+* **Questions**
+* **Scoring**
+* **Candidates**
+* **Live Session**
+
+.. image:: survey_essentials/survey-options-tab.png
+   :align: center
+   :alt: options tab on survey template form
+
+Questions
+---------
+
+This section primarily focuses on the overall presentation of your survey.
+
+First, you select the layout. You can choose from:
+
+* *One page with all the questions*
+* *One page per section*
+* *One page per question*
+
+You can also choose how you want to display the participant's progress during the survey. It can
+be shown as either a *Percentage* or a *Number*.
+
+Below that, you can choose to put a time limit on your survey, modify the question selection, and
+decide if you'd like to present your participants with a *Back Button*.
+
+.. image:: survey_essentials/questions-setting-section.png
+   :align: center
+   :alt: questions section of survey settings
+
+.. seealso::
+    - :doc:`time_random`
+
+Scoring
+-------
+
+In this section, you decide how you want the survey to be scored. You can choose from the following:
+
+* *No scoring*
+* *Scoring with answers at the end*
+* *Scoring without answers at the end*
+
+If you select either *Scoring with answers at the end* or *Scoring without answers at the end*,
+two more fields appear. In which, you decide what the "Success %" is, and you can choose to make
+this a certification.
+
+If you check the box next to "Is a certification," three more additional fields appear. You can
+pick an email template, certification template, and choose to give participants a badge for
+successfully completing the survey.
+
+And, if you check the box next to "Give Badge," you are able to select which Certification Badge
+you want to give to your participants.
+
+.. image:: survey_essentials/survey-scoring-section.png
+   :align: center
+   :alt: scoring section of survey settings
+
+.. seealso::
+    - :doc:`scoring`
+
+Candidates
+----------
+
+In this section, you can decide who can access this survey.
+
+The "Access Mode" allows you to choose between *Anyone with the link* and *Invited people only*.
+You can also decide if you want to grant access to *Appraisal Managers Only*.
+
+You are also able to decide if you want to require a login to participate in this survey here, as
+well.
+
+If you check the box next to "Attempts Limit," a new field appears, in which you will enter in
+how many attempts you are willing to give to your participants.
+
+.. image:: survey_essentials/candidates-section.png
+   :align: center
+   :alt: candidates section of survey settings
+
+Live Session
+------------
+
+This section is dedicated to users who are conducting Live Session surveys, wherein they directly
+engage with an audience at the same time - much like a live event.
+
+Here, you can choose to reward participants for quick answers, as well as customize the specific
```

Comment:
I would specify that rewarding participants for quick answers give them more points. If possible, I would go further and give an example of how this translate in reality.